yar/yar.php

```diff
@@ -98,7 +98,7 @@ public function __call ( $method , $parameters ){}
      * - YAR_OPT_CONNECT_TIMEOUT
      * - YAR_OPT_HEADER (Since 2.0.4)
      * @param $value
-     * @return object|false Returns $this on success or FALSE on failure.
+     * @return $this|false Returns $this on success or FALSE on failure.
```

Comment:
Please change $this to class name

Reply:
But ```$this``` or ```static``` is more correct here, or? 

```php
<?php

/* // server.php
<?php

class API {
    public function test(int $i) {
        return $i + 1;
    }
}

$service = new Yar_Server(new API());
$service->handle();
*/

class Yar_Client2 extends Yar_Client
{
    /**
     * @return $this
     */
    public function lall(): self {
        return $this;
    }
}

$client = new Yar_Client2("http://localhost/server.php");

echo $client->setOpt(YAR_OPT_CONNECT_TIMEOUT, 10)->lall()->test(99); // output: 100
```

Reply:
The same answer as above actually. If a class can be extended it makes sense to use `static` as a return type as `$this` isn't actually a type

Reply:
There are already many stubs with "$this" in phpdocs, should we convert all to "static"? Should I create a separate PR for that?

Reply:
Yes, it makes sense and yes, please create a separate PR for this. Thank you for your contribution!